Formula/lasso.rb

```diff
@@ -21,17 +21,14 @@ class Lasso < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "python@3.10" => :build
-  depends_on "six" => :build
```

Comment:
```
ModuleNotFoundError: No module named 'six'
```

Seems like it is installed on macOS. Do we know if scripts can be run on Python 2 and if macOS `/usr/bin/python` has `six` already installed?

Reply:
It does. The only issue is that Monterey does not have `/usr/bin/python`.

Reply:
They do check for `python3` first (https://repos.entrouvert.org/lasso.git/tree/configure.ac#n136)

Just wanted to make sure for Mojave and older since we don't test those on CI.